target/linux/mvebu/cortexa9/base-files/etc/uci-defaults/03_wireless

```diff
@@ -37,16 +33,8 @@ linksys,wrt32x)
 		;;
 	esac
 
-	case "$board" in
-		linksys,wrt1900ac-v1)
-			WIFIMAC0=$WIFIMAC2G
-			WIFIMAC1=$WIFIMAC5G
-		;;
-		*)
-			WIFIMAC0=$WIFIMAC5G
-			WIFIMAC1=$WIFIMAC2G
-		;;
-	esac
+	WIFIMAC0=$WIFIMAC2G
+	WIFIMAC1=$WIFIMAC5G
```

Comment:
Since the case is gone, this reassignment actually isn't needed anymore either, and one could just use WIFIMAC2G/WIFIMAC5G in the uci commands below directly.